Navigation/nav.html

```diff
@@ -0,0 +1,168 @@
+               </ul>
+         
+       </div>
+      </nav>
+
+
+
+    <br> <br>
+
+
+      <div class="parent-container">
+
+    
+        <div class="slidebar">
+
+          <ul class="sub-topic-list" > 
+                <h2 >Components</h2>
+        
+                <li class="list-links"> <a href="../Avatar/avatar.html" >   Avatar</li> </a>
+                <li class="list-links"> <a href="../Alert/alert.html" >  Alert</li> </a>
+                <li class="list-links"> <a href="../Badge/badge.html" >  Badge</li> </a>
+                <li class="list-links"> <a href="../Button/button.html" >  Button</li> </a>
+                <li class="list-links"> <a href="../Cards/cards.html" >  Cards</li> </a>
+                <li class="list-links"> <a href="../Image/image.html" >  Image</li> </a>
+                <li class="list-links"> <a href="../Input/input.html" >  Input</li> </a>
+                <li class="list-links"> <a href="../Lists/lists.html" >  List</li> </a>
+                <li class="list-links"> <a href="../Rating/rating.html" > Rating</li> </a>
+                <li class="list-links"> <a href="../Toast/toast.html" > Toast</li> </a>
+                <li class="list-links"> <a href="../Typography/typography.html" >  Typography</li> </a>
+                <li class="list-links"> <a href="../Navigation/nav.html" >  Navigation</li> </a>
+            </ul>
+          </div>
+      
+      
+        <div class="doc-content">
+         <div class="content-title" > Navigation</div>
+         
+         <br> <br>
+         <div class="content-subtitle" > 
+    
+            Nav bar displays the list of choices for the user avaible of the web page.
+            </div>
+ <hr>
+             
+             
+             
+         <!-- <div class="circle-avatar"> -->
+            <div class="container">
+                <nav class="nav__bar">
+                  <div >
+                    <div class="left-side-link">
+                      <a class="nav-a-link" href="../index.html" > Geeta UI</a>
+                     </div>
+                  </div>
+                 
+                  <div class="nav-section right-side-links">
+          
+                      <ul> 
+                          <li class="nav-li-links">
+                            <span >  <a class="nav-a-link" href="../index.html" >Home</a> </span>
+                          </li>
+               
+                           <li  class="nav-li-links">
+                            <span > <a class="nav-a-link" href="../Docs/docs.html"> Docs</a>   </span>
+                           </li>
+               
+               
+                           <li  class="nav-li-links">   <span >  <button class="theme-button" id="theme-btn" > 
+                              <i class='fas fa-moon'> </i>   </button>
+                           </li>
+                       </ul>
+                 
+               </div>
+              </nav> 
+           </div> <br> 
```

Comment:
again try not to add br tag for margins.
https://webmasters.stackexchange.com/questions/79199/why-isnt-the-html-break-tag-acceptable-anymore-for-top-and-bottom-margins
check this out.